espn_api/football/league.py

```diff
@@ -398,3 +399,63 @@ def message_board(self, msg_types: List[str] = None):
             for msg in msgs:
                 messages.append(msg)
         return messages
+    
+
+    
+    def draft_evaluation(self):
+        #Calculating draft quality score based on projected score vs actual score for each pick
+        draftData = self.draft
+        #Process of removing any duplicates from draft data for accuracy
+        duplicates = set()
+        duplicates_removed = []
+        for pick in draftData:
+            pick_id = (pick.round_num, pick.round_pick)
+            if pick_id not in duplicates:

```

Comment:
Would there ever be duplicates in this data?

Reply:
When I printed out league.draft, the draft picks would repeat so that the set was double the actual size, sometimes even triple. I couldn't find why, so I decided to just filter it out with this method.